src/LiveComponent/doc/index.rst

```diff
@@ -931,14 +931,18 @@ a class:
     <!-- add multiple classes when loading -->
     <div data-loading="addClass(opacity-50 disabled)">...</div>
 
-Sometimes you may want to add or remove an attribute when loading. That
+Sometimes you may want to add or remove an HTML attribute when loading. That
 can be accomplished with ``addAttribute`` or ``removeAttribute``:
 
 .. code-block:: html+twig
 
     <!-- add the "disabled" attribute when loading -->
     <div data-loading="addAttribute(disabled)">...</div>
 
+    <!-- this feature only works for empty attributes that don't define any value;
```

Comment:
What about wrapping this in a caution annotation? I'm worried readers might skip over _the following example won't work_ and think it's valid

Reply:
I agree - excellent detail to point out